### ISSUE TITLE
A2. River network as overlay

### ISSUE THREAD

**Issue by pgiraud** [GFDRR/thinkhazard#680]
As a quick test, I added a layer showing the waterways from a mapbox tileset.

![screenshot from 2016-11-28 17 28 42](https://cloud.githubusercontent.com/assets/319774/20676648/ba03ff46-b590-11e6-8d1d-68e15e83a535.png)

In my opinion, there's no real need to have a toggle button for showing a river network in the map for flood hazardtype. The waterways could be added to the currently used background layer in addition to the water areas.

**Comment by stufraser1** [comment]
OK, this looks to be a good solution - so if I understand, this addition to the basemap could be shown for river and urban flood only, and basemap for other hazards would be shown without this detail?

Secondly, is it possible to include a shaded areas to indicate urban land, in this same way? This will be important for urban flood map, and may be useful in other hazards too for users to orient themselves to urban extents

**Comment by pgiraud** [comment]
> OK, this looks to be a good solution - so if I understand, this addition to the basemap could be shown for river and urban flood only, and basemap for other hazards would be shown without this detail?

In my opinion, the river network doesn't hurt. It could be displayed for any hazard type.

> Secondly, is it possible to include a shaded areas to indicate urban land, in this same way? This will be important for urban flood map, and may be useful in other hazards too for users to orient themselves to urban extents

I'm pretty sure there's data for that in MapBox.

@vdeparday @ingenieroariel Can someone at GFDRR try to add river network and urban land (residential areas) to the MapBox layer? Or give me the credentials?
It's probably a good idea to the appropriate changes on a copy in order not to break the current layer.

**Comment by ingenieroariel** [comment]
Pierre,

That's done on my personal mapbox account (free tier) and we have not hit
the limits. Best may be for you to set up a camp2camp account, do it there
and then change the map id in the javascript interface.

**Comment by pgiraud** [comment]
@ingenieroariel Can you please send me the style config? You should be able to download the style from the mapbox styles interface.

**Comment by ingenieroariel** [comment]
Unfortunately not anymore, Mapbox changed their UI and what we used at the time is now deprecated.

<img width="1030" alt="screen shot 2016-12-22 at 5 37 13 pm" src="https://cloud.githubusercontent.com/assets/54999/21442675/67808420-c86d-11e6-89cd-4d4b9139bc81.png">

But it was not hard to create (less than 20 mins) and no one is attached to that particular version.

**Comment by pgiraud** [comment]
@ingenieroariel Can you please try to connect to https://www.mapbox.com/studio/styles/ and confirm that you are not able to export (or download) the style used for generating the tileset we are currently using on ThinkHazard! ?
It would be really helpful.
Thanks.

**Comment by ingenieroariel** [comment]
TLDR: I have the tileset but not the style.
<img width="968" alt="screen shot 2017-01-29 at 8 33 21 pm" src="https://cloud.githubusercontent.com/assets/54999/22410156/392b0588-e662-11e6-8ec0-567f3849f6fe.png">

I can confirm it is not possible.

I created the style very early in mapbox vector tile support and through their online UI. Since I did not know the original data or code, chances are they only kept a copy of it but not a way to reproduce it without the (now deprecated) tools.

**Comment by pgiraud** [comment]
@stufraser1 We now use a new base map layer as per #678.
I hoped that in this new layer the river network would be more visible. It's unfortunately not the case.
I suppose that you are still interested to make the river network more visible when showing water related hazard types (River/Urban flood). If so, can you please ask the team responsible for the mapbox tilesets to provide a new set with the rivers only that we could integrate and put on top of the base map?

**Comment by stufraser1** [comment]
I've created a Mapbox style for use in TH!
It is on the free account, so only up to 50k views, but I hope to have this duplicated on the WB account before we exceed that.

Please update the basemap on int server as follows
Style:   mapbox://styles/stufraser1/cj3g19csf00062sjso7dfj4ns
Access token:    pk.eyJ1Ijoic3R1ZnJhc2VyMSIsImEiOiJQdnhvZTlnIn0.SEC9tGQDtw9yPQssVyF-8Q

**Comment by tonio** [comment]
I can't get it to work. Do you have an example page or a more complete url to test ? — url is tied to api version used —.
Thanks in advance.

**Comment by stufraser1** [comment]
URL
https://api.mapbox.com/styles/v1/stufraser1/cj3g19csf00062sjso7dfj4ns.html?title=true&access_token=pk.eyJ1Ijoic3R1ZnJhc2VyMSIsImEiOiJQdnhvZTlnIn0.SEC9tGQDtw9yPQssVyF-8Q#8.5/29.710234/31.063081/0

**Comment by tonio** [comment]
Fixed & deployed to integration.

**Comment by fjacon** [comment]
@stufraser1 can we close this ticket ?

**Comment by stufraser1** [comment]
Confirmed included and visible. 
One problem: Admin boundaries (not the one selected) are shown in a think blue line, which is now confused with rivers - e.g., http://int.thinkhazard.org/en/report/18373-italy-piemonte-alessandria. 
Possible to change these unselected admin boundaries to a thin black line, for example, for better distinction?
If actioned, can be closed.

**Comment by stufraser1** [comment]
I will solve admin boundaries in a new layer. Satisfied with river overlay though. Closing
